net/kea/Makefile

```diff
@@ -0,0 +1,198 @@
+#
+# Copyright (C) 2019 Banglang Huang <banglang.huang@foxmail.com>
+# Copyright (C) 2019 Rosy Song <rosysong@rosinson.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=kea
+PKG_VERSION:=460-update-mysql-database-schema-for-dhcpv6-config-backend--port_base
```

Comment:
This is not only a beta version but also a ridiculous name for it. The latest stable version is 1.5.0.